browser/ui/views/sidebar/sidebar_container_view.cc

```diff
@@ -109,6 +114,33 @@ void SidebarContainerView::UpdateSidebar() {
   sidebar_control_view_->Update();
 }
 
+void SidebarContainerView::ShowCustomContextMenu(
+    const gfx::Point& point,
+    std::unique_ptr<ui::MenuModel> menu_model) {
+  // Show context menu at in screen coordinates.
+  gfx::Point screen_point = point;
+  ConvertPointToScreen(sidebar_panel_view_, &screen_point);
+  context_menu_model_ = std::move(menu_model);
+  context_menu_runner_ = std::make_unique<views::MenuRunner>(
+      context_menu_model_.get(),
+      views::MenuRunner::HAS_MNEMONICS | views::MenuRunner::CONTEXT_MENU);
+  const int active_index = sidebar_model_->active_index();
+  if (active_index == -1) {
+    LOG(ERROR) << __func__
```

Comment:
should this be a VLOG?

Reply:
I think it's error state because `-1` means any sidebar panel is not opened.
In that situation, getting context menu showing request is weird.

Reply:
This could not be happened in normal situation.
and I got this when I load sidebar-bookmarks webui url in the tab.

Reply:
maybe just make it as a comment, because it looks like a valid event and we just should handle it and return?